StandardCyborgUI/StandardCyborgUI/PointCloudCommandEncoder.metal

```diff
@@ -49,7 +49,7 @@ vertex ProjectedVertex RenderSCPointCloudVertex(Vertex v [[stage_in]],
     float pointSize = (normal.z > 0 && v.weight > 0.0) ? (uniforms->pointSize * fadeIn) : 0;
     
     // Eye vector is implicitly at the origin since we're in view coordinates
-    float3 positionRelEye = -normalize(viewPosition);
+    float3 positionRelEye = normalize(viewPosition);
```

Comment:
I believe it was a bit suspicious that a negative sign was here to begin with. Without this you get the shiny matcap look.